app/top-page.module.css

```
@@ -4,83 +4,120 @@
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+
+  background-image: linear-gradient(
+    to bottom right,
+    rgba(173, 185, 252, 0) 0%,
+    rgba(227, 231, 255, 0.8) 40%,
+    rgb(255, 255, 255) 100%
+  );
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)